### PR TITLE
Option on metrics histograms to show count

### LINKED ISSUE #965: Metrics: Option to display histogram values as counts
Aspire only displays a percentile chart for histograms. This is a problem because there is no way to view how many values were recorded to histograms over time.

OTEL semantic conventions for HTTP only have a duration histogram and active_requests up/down counter, e.g. `http.client.request.duration`. HTTP doesn't have total counters for the number of requests. The reason is that a histogram tracks values and counts.

Aspire metrics UI needs a way to show counts for a histogram. UI and filtering capabilities should be the same as other count charts.

There are a couple of options:

1. A button to switch between a percentile chart and a count chart
2. Display the percentile chart and count chart at the same time
3. Add count to the percentile chart as another series. [Maybe a dashed line?](https://plotly.com/javascript/line-charts/#line-dash) One chart displays both sets of information at the same time.

1 and 2 can reuse the existing count chart. 3 is harder because it means new plotly configuration, which is easier said than done.

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes https://github.com/dotnet/aspire/issues/965

Adds a switch to histogram charts to display the count of the instrument, e.g. the number of HTTP requests.

Demo:
metrics-show-count

### CODE REVIEW

File: src/Aspire.Dashboard/Resources/ControlsStrings.resx [hunk 1]
```diff
@@ -127,8 +127,7 @@
     <value>Select filters</value>
   </data>
   <data name="EnvironmentVariablesFilterToggleShowSpecOnly" xml:space="preserve">
-    <value>Show Spec Only</value>
-    <note>Spec means from the resource specification</note>
```
[the remark this answers is not in the document]
Comment (drewnoakes) [reply src/Aspire.Dashboard/Resources/ControlsStrings.resx:131]: Looks like a bug in an internal preview feature. Paging @melytc and @haileymck to investigate.

**Review by drewnoakes — COMMENTED** [review]
Looks good, other than the regression in the resx. Approving on the basis that it would be reverted before merge.

**Review by drewnoakes — APPROVED** [review]

File: src/Aspire.Dashboard/Resources/ControlsStrings.resx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JamesNK) [reply src/Aspire.Dashboard/Resources/ControlsStrings.resx:131]: Also happens in int preview

[the remark this answers is not in the document]
Comment (JamesNK) [reply src/Aspire.Dashboard/Resources/ControlsStrings.resx]: https://github.com/dotnet/aspire/issues/1287

**Pull request merged**